src/logic/wallets/tests/ethAddresses.test.ts

```diff
@@ -0,0 +1,281 @@
+    // given
+    const safeAddress = '0xdfA693da0D16F5E7E78FdCBeDe8FC6eBEa44f1Cf'
+    const safeAddress2 = '0x344B941b1aAE2e4Be73987212FC4741687Bf0503'
+
+    // when
+    const result = sameAddress(safeAddress, safeAddress2)
+
+    // then
+    expect(result).toBe(false)
+  })
+})
+
+describe('shortVersionOf', () => {
+  it('Given no address, returns Unknown ', () => {
+    // given
+    const safeAddress = null
+    const cut = 5
+    const expectedResult = 'Unknown'
+
+    // when
+    const result = shortVersionOf(safeAddress, cut)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given 0x344B941b1aAE2e4Be73987212FC4741687Bf0503 and a cut = 5, returns 0x344...f0503', () => {
+    // given
+    const safeAddress = '0x344B941b1aAE2e4Be73987212FC4741687Bf0503'
+    const cut = 5
+    const expectedResult = `0x344...f0503`
+
+    // when
+    const result = shortVersionOf(safeAddress, cut)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given a cut value bigger than the address length, returns the same address', () => {
+    // given
+    const safeAddress = '0x344B941b1aAE2e4Be73987212FC4741687Bf0503'
+    const cut = safeAddress.length
+    const expectedResult = safeAddress
+
+    // when
+    const result = shortVersionOf(safeAddress, cut)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+})
+
+describe('isUserAnOwner', () => {
+  it('Given no safe, returns false ', () => {
+    // given
+    const userAddress = 'address1'
+    const safeInstance = null
+    const expectedResult = false
+
+    // when
+    const result = isUserAnOwner(safeInstance, userAddress)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given no userAccount, returns false ', () => {
+    // given
+    const userAddress = null
+    const owners = List([makeOwner({ address: userAddress })])
+    const safeInstance = makeSafe({ owners })
+    const expectedResult = false
+
+    // when
+    const result = isUserAnOwner(safeInstance, userAddress)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given a safe without owners, returns false ', () => {
+    // given
+    const userAddress = 'address1'
+    const owners = null
+    const safeInstance = makeSafe({ owners })
+    const expectedResult = false
+
+    // when
+    const result = isUserAnOwner(safeInstance, userAddress)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given a safe with 1 owner, and an userAddress equals to that owner, returns true ', () => {
+    // given
+    const userAddress = 'address1'
+    const owners = List([makeOwner({ address: userAddress })])
+    const safeInstance = makeSafe({ owners })
+    const expectedResult = true
+
+    // when
+    const result = isUserAnOwner(safeInstance, userAddress)
+
+    // then
+    expect(result).toBe(expectedResult)
+  })
+  it('Given a safe with 1 owner, and an userAddress different to that owner, returns true ', () => {
```

Comment:
```suggestion
  it('Given a safe with 1 owner, and a userAccount different to that owner, returns false', () => {
```

Anyway, I would have written it differently. Something around:

```js
describe('isUserAnOwner', () => {
  it("Should return false if there's no Safe")
  it("Should return false if there's no `userAccount`")
  it("Should return false if there are no owners for the Safe") // this one is strange, to say the least
  it("Should return true if `userAccount` is not in the list of Safe's owners")
  it("Should return false if `userAccount` is not in the list of Safe's owners")
})
```

I like the `it` -> `should...` writing.

Reply:
I really like your approach, will do those changes thanks!

Reply:
I also like the it -> should writing for test titles.

Would leave the Given / When / Then for the comments inside